sdk/monitor/azure-monitor-opentelemetry-exporter/README.md

```diff
@@ -383,6 +383,11 @@ gauge = meter.create_observable_gauge("gauge", [observable_gauge_func])
 #### Metric custom views
 
 ```python
+"""
+This example shows how to customize the metrics that are output by the SDK using Views. Metrics created
+and recorded using the sdk are tracked and telemetry is exported to application insights with the
+AzureMonitorMetricsExporter.
+"""
```

Comment:
It seems some examples have these explanation comments IN CODE while others have explanation beforehand. I like the latter. What do you think?

Reply:
Fixed to be consistent.